main.py

```diff
@@ -48,177 +62,352 @@ def validate_yaml_file(yaml_path: Path) -> dict:
-        return secrets['llm_api_key']
 
 class FileManager:
+    """Class for handling file operations related to the application."""
+
     @staticmethod
     def validate_data_folder(app_data_folder: Path) -> tuple:
+        """
+        Validates that the required files exist in the data folder.
+        """
         if not app_data_folder.exists() or not app_data_folder.is_dir():
             raise FileNotFoundError(f"Data folder not found: {app_data_folder}")
 
         required_files = [SECRETS_YAML, WORK_PREFERENCES_YAML, PLAIN_TEXT_RESUME_YAML]
         missing_files = [file for file in required_files if not (app_data_folder / file).exists()]
         
         if missing_files:
-            raise FileNotFoundError(f"Missing files in the data folder: {', '.join(missing_files)}")
+            raise FileNotFoundError(
+                f"Missing files in the data folder: {', '.join(missing_files)}"
+            )
 
-        output_folder = app_data_folder / 'output'
+        output_folder = app_data_folder / "output"
         output_folder.mkdir(exist_ok=True)
-        return (app_data_folder / SECRETS_YAML, app_data_folder / WORK_PREFERENCES_YAML, app_data_folder / PLAIN_TEXT_RESUME_YAML, output_folder)
+        return (app_data_folder / SECRETS_YAML, app_data_folder / WORK_PREFERENCES_YAML, app_data_folder / PLAIN_TEXT_RESUME_YAML, output_folder,)
 
     @staticmethod
-    def file_paths_to_dict(resume_file: Path | None, plain_text_resume_file: Path) -> dict:
+    def file_paths_to_dict(
+        resume_file: Optional[Path], plain_text_resume_file: Path
+    ) -> dict:
+        """
+        Returns a dictionary containing paths to the resume and plain text resume files.
+        """
         if not plain_text_resume_file.exists():
-            raise FileNotFoundError(f"Plain text resume file not found: {plain_text_resume_file}")
+            raise FileNotFoundError(
+                f"Plain text resume file not found: {plain_text_resume_file}"
+            )
 
-        result = {'plainTextResume': plain_text_resume_file}
+        result = {"plainTextResume": plain_text_resume_file}
 
         if resume_file:
             if not resume_file.exists():
                 raise FileNotFoundError(f"Resume file not found: {resume_file}")
-            result['resume'] = resume_file
+            result["resume"] = resume_file
 
         return result
 
+
 def init_browser() -> webdriver.Chrome:
+    """Initializes the Chrome browser with specified options."""
     try:
         options = chrome_browser_options()
         service = ChromeService(ChromeDriverManager().install())
         return webdriver.Chrome(service=service, options=options)
     except Exception as e:
         raise RuntimeError(f"Failed to initialize browser: {str(e)}")
 
-def create_and_run_bot(parameters, llm_api_key):
+
+def create_and_run_bot(email, password, parameters, llm_api_key):
```

Comment:
Email and password are not needed to provide. bot prompts screen to user for then to enter creds & clear any things like capcha

Reply:
done